src/libraries/System.Net.Http/src/System/Net/Http/Metrics/MetricsHandler.cs

```diff
@@ -209,6 +221,14 @@ private static object GetBoxedStatusCode(int statusCode)
                 : statusCode;
         }
 
+        private static string GetStatusCodeString(int statusCode)
+        {
+            string[] strings = LazyInitializer.EnsureInitialized(ref s_statusCodeStrings, static () => new string[512]);
```

Comment:
Would it make sense to save size by offsetting indexes by 400?
Likely not needed, but perhaps in future? (we would have to change the method to `GetStatusCode4xx5xxString`)